src/test/e2e/tests/registerApiViaCICD.test.ts

```diff
@@ -1,37 +1,38 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
+
 import { expect, test } from '../baseTest';
+import { APICenter, Timeout, VSCode } from '../utils/constants';
+import VscodeOperator from '../utils/vscodeOperator';
 
 test('trigger generateAPIviaCICD with Azure DevOps', async ({ workbox }) => {
+    await workbox.waitForTimeout(Timeout.PREPARE_TEST);
     // wait API Center extension installed on VS Code.
-    await workbox.getByRole("tab", { name: "API Center" }).isVisible();
-    await workbox.getByRole("tab", { name: "API Center" }).locator('a').click();
+    expect(await VscodeOperator.isSideTabItemExist(workbox, VSCode.TAB_API_CENTER)).toBeTruthy();
```

Comment:
it's better to replace the VSCode.TAB_API_CENTER to specific string, we need to assert the workbox with the string, not config string assert to itselft.